### PR TITLE
File Added : Assignments 5 and 6.

### PR DESCRIPTION
**Pull request by yogesh141293**

### CODE REVIEW

**Review by SamikshaWani — APPROVED** [review]

**Review by Oshin-250197 — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

File: Yogesh_Ghodmare/HTML/Assig5/Que.1.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <tr>
+            <td>
+                <span class="titlet">&nbsp;Yogesh S.Ghodmare</h3>
+            </td>
+        </tr>
+        <tr class="bgcolor1">
+            <td>
+                <h2 class="right"><i>About Me&nbsp;&nbsp;</i></h2>
+                <div class="shadow"
+                    style="background-color:#b0c4de; background-size: cover; cover;width:96%;padding:7px;">
```
Comment (SamikshaWani) [inline Yogesh_Ghodmare/HTML/Assig5/Que.1.html]: add space after : (width,padding)

File: Yogesh_Ghodmare/HTML/Assig6/Form_effect.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+header{
+    text-align: right;
+    background-color: rgb(83, 83, 90);
+    height: 100%;
+    width: 100%;
+    border-radius: 5px 10px;
+}
+h3{
```
Comment (SamikshaWani) [inline Yogesh_Ghodmare/HTML/Assig6/Form_effect.css]: space in between header {
also in h3 {

File: Yogesh_Ghodmare/HTML/Assig6/Form_effect.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    text-align: right;
+    background-color: rgb(130, 182, 202);
+    border-radius: 5px 10px;
+}
```
Comment (SamikshaWani) [inline Yogesh_Ghodmare/HTML/Assig6/Form_effect.css]: add extra line

File: Yogesh_Ghodmare/HTML/Assig6/Que.1.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <body>
+        <div>
+            <header>
+                <h2>MI-9 Intern HTML Assesment</h2>
+            </header>
+            <br /><br />
+            <hr />
+            <h3><i>Presonel Details</i></h3>
+            <form action="">
+                <label for="fname">Fist Name<input type="text" placeholder="Fist Name" /></label>
```
Comment (SamikshaWani) [inline Yogesh_Ghodmare/HTML/Assig6/Que.1.html:19]: placeholder second word first letter should be small

File: Yogesh_Ghodmare/HTML/Assig6/Que.1.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+                <br /><br />
+                <label for="lname">Last Name<input type="text" placeholder="Last Name" /></label><br /><br />
```
Comment (SamikshaWani) [inline Yogesh_Ghodmare/HTML/Assig6/Que.1.html:21]: placeholder second word first letter should be small

File: Yogesh_Ghodmare/HTML/Assig6/Form_effect.css (hunk 3 above)
Comment (yogesh141293) [reply Yogesh_Ghodmare/HTML/Assig6/Form_effect.css]: Changes Done

File: Yogesh_Ghodmare/HTML/Assig6/Que.1.html (hunk 5 above)
Comment (yogesh141293) [reply Yogesh_Ghodmare/HTML/Assig6/Que.1.html:21]: Changes Done

File: Yogesh_Ghodmare/HTML/Assig5/Que.1.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>home</title>
+    <meta name="description" content="Home">
+    <meta name="keywords" content="">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1" />
+    <link rel="stylesheet" href="Que.1.css">
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig5/Que.1.html]: indentation error

File: Yogesh_Ghodmare/HTML/Assig5/Que.1.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                        <li><a href="#hobbies" target="_blank">My Hobbies</a></li>
+                        <li><a href="#photo" target="_blank">Photo Gallery</a></li>
+                    </ul>
+                </div>
+                </div>
+            </td>
+        </tr>
+    </table>
+   </body>
+</html>
```
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig5/Que.1.html:43]: indentations error

File: Yogesh_Ghodmare/HTML/Assig6/Form_effect.css (hunk 3 above)
Comment (Pawan1005) [inline Yogesh_Ghodmare/HTML/Assig6/Form_effect.css]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**